### PR TITLE
Introducing Gazebo support for Staubli TX90 family

### PR DESCRIPTION
**Pull request by MuriloMartins**
Adding Gazebo-related files to simulate the Staubli TX90 family.

Please note that this PR depends on PR #5 .

### CODE REVIEW

**Comment by MuriloMartins** [comment]
@gavanderhoorn I split the old PR into two PRs.

I got things a bit muddled up, but I think it looks tidy now...
(I need to tidy up our fork now)

File: staubli_tx90_gazebo/urdf/tx90_macro.xacro [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    <actuator name="${prefix}motor6">
+      <mechanicalReduction>1</mechanicalReduction>
+    </actuator>
+  </transmission>
+  <!-- end of transmission list -->
+
+  <!-- Gazebo-specific link properties -->
+  <gazebo reference="${prefix}base_link">
+    <material>Gazebo/Yellow</material>
+    <turnGravityOff>true</turnGravityOff>
```
Comment (gavanderhoorn) [inline staubli_tx90_gazebo/urdf/tx90_macro.xacro:75]: We may have discussed this before, but why is `turnGravityOff` set to `true` here?

**Comment by MuriloMartins** [comment]
I cannot remember exactly (it is disabled in the xacro files for the RX160 models).

I vaguely remember something related to testing, perhaps before I had the masses, coms, and inertia matrices from SolidWorks.

**Comment by MuriloMartins** [comment]
@gavanderhoorn leave it consistent with RX160 models for now?

**Comment by gavanderhoorn** [comment]
Yes, that would make sense for now.

File: staubli_tx90_gazebo/launch/tx90_gazebo.launch [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="paused" default="false"/>
+  <arg name="gui" default="true"/>
+  <arg name="remap" default="false"/>
```
Comment (gavanderhoorn) [inline staubli_tx90_gazebo/launch/tx90_gazebo.launch:5]: Just wondering: should we perhaps rename this argument to something like `rosi_compat` or `rosind_compat`?

Comment (MuriloMartins) [inline staubli_tx90_gazebo/launch/tx90_gazebo.launch:5]: Sounds good!
Shall we do that for the RX160 family too?
I'll address this soon-ish.

**Comment by gavanderhoorn** [comment]
@MuriloMartins: did you still want to make some changes to this PR?

If not, I can merge it in manually, as I've done with #5.

**Comment by MuriloMartins** [comment]
Hi @gavanderhoorn !
No I don't have any changes in mind for this PR at the moment.
Perhaps just renaming the argument... would it be OK to keep this as it is and then I can rename the argument for both TX90 and RX160 families on a separate PR (to keep both robot models consistent)?

**Comment by gavanderhoorn** [comment]
@MuriloMartins wrote:

> Perhaps just renaming the argument... would it be OK to keep this as it is and then I can rename the argument for both TX90 and RX160 families on a separate PR (to keep both robot models consistent)?

yes, that makes sense. See #7.

I'll put this PR on my merge-TODO list then.

**Comment by gavanderhoorn** [comment]
@MuriloMartins: just merged this manually (to resolve the conflicts after I merged #5). See 5d12ff48.

Thanks again for contributing, much appreciated :).

**Pull request closed without merging**